### DISCUSSION TITLE
Ability to Play/Pause Media?

### DISCUSSION THREAD

**Discussion by Darce256** [Palakis/obs-websocket#1072] [Q&A]
I have been looking through the websocket documentation (I am using the websocket-js lib) and I was hoping to find a way to start/stop a media item.

Currently I can enable a hidden media item and have it play once but then I would need to disable and then re-enable to reproduce the functionality - I figure the ideal way would be to just play the media while enabled rather than enable/disable to achieve the desired effect.

Maybe I missed the option?

Thanks!

**Comment by tt2468** [comment]
`TriggerMediaInputAction`
